### ISSUE TITLE
`stable` is dual-tagged

### ISSUE THREAD

**Issue by bowerscd** [factoriotools/factorio-docker#511]
Per readme, `stable` is both tagged as 2.0.8 and 2.0.7. This makes stable point to `2.0.7`:

```logs
~ [1]> sudo docker image inspect factoriotools/factorio:stable
[
    {
        "Id": "sha256:25c36738ce940c9adb4401974d19b32b886dedc144ef883091c27501873a64b1",
        "RepoTags": [
            "factoriotools/factorio:stable"
        ],
        "RepoDigests": [
            "factoriotools/factorio@sha256:6541b6a3f0b4683ad33c2e79d0ed24004d5d5f8697e758488bac5593fc8913ce"
        ],
        "Parent": "",
        "Comment": "buildkit.dockerfile.v0",
        "Created": "2024-10-21T21:22:42.19898242Z",
        "DockerVersion": "",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "ExposedPorts": {
                "27015/tcp": {},
                "34197/udp": {}
            },
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "PORT=34197",
                "RCON_PORT=27015",
                "SAVES=/factorio/saves",
                "PRESET=",
                "CONFIG=/factorio/config",
                "MODS=/factorio/mods",
                "SCENARIOS=/factorio/scenarios",
                "SCRIPTOUTPUT=/factorio/script-output",
                "PUID=845",
                "PGID=845",
                "VERSION=2.0.7",
                "SHA256=e6a64a9ee5b1d3773cd1f40aa5cf78e91960eade6c508fd365c34752afdf2762"
            ],
            "Cmd": null,
            "Image": "",
            "Volumes": {
                "/factorio": {}
            },
            "WorkingDir": "",
            "Entrypoint": [
                "/docker-entrypoint.sh"
            ],
            "OnBuild": null,
            "Labels": {
                **"factorio.version": "2.0.7",**
                "maintainer": "https://github.com/factoriotools/factorio-docker"
            },
...
```

When stable is, in fact, [`2.0.8`](https://www.factorio.com/download).

**Comment by Hhovland** [comment]
Until fixed if you need to force a server up to date use factoriotools/factorio:stable-2.0.8 as your server version 
(This fixed our server temporarily.

**Comment by Fank** [comment]
I removed the 2.0.7 release for now, the image on docker will be still available but not build.
This is just a temporary workaround, and still needs to be fixed.

**Comment by Jayden-Lind** [comment]
Just using `factoriotools/factorio:2.0` should suffice as well

**Comment by Fank** [comment]
I hope I fixed it with #515
